frontend/src/metabase/ui/components/data-display/TreeTable/TreeTableHeader/HeaderCell/HeaderCell.module.css

```diff
@@ -0,0 +1,19 @@
+.pill {
+  border-radius: 6px;
+  background-color: color-mix(in srgb, var(--mb-color-brand), transparent 92%);
```

Comment:
We should avoid doing `color-mix` in css modules. In this case, we likely want `background-brand`